### PR TITLE
Support UDP Transport Server

### LINKED ISSUE #1573: udp in Transportserver crd
#### Title
Support UDP in Transportserver

#### Description
Add support for UDP to match TCP in transport server.  I use backend DNS Servers for instance on tcp and udp

#### Actual Problem
Currently tcp is only supported so I cannot move to crd design.

#### Solution Proposed
Allowed udp same as tcp

### PR DESCRIPTION
**Pull request by sravyap135**
**Description**:  Support for type UDP Transport Server

**Changes Proposed in PR**: Added support for UDP Transport Server in crmanager 

**Fixes**: This PR resolves #1573 

## General Checklist
- [x] Updated Added functionality/ bug fix in release notes
- [x] Added examples for new feature
- [x] Updated the documentation

## CRD Checklist
- [x] Updated required CR schema

### CODE REVIEW

**Review by vklohiya — APPROVED** [review]
Looks Fine

**Review by agupta49 — APPROVED** [review]

File: pkg/crmanager/validate.go [hunk 1]
```diff
@@ -92,5 +92,12 @@ func (crMgr *CRManager) checkValidTransportServer(
 		return false
 	}
 
+	if len(tsResource.Spec.Type) == 0 {
```
Comment (agupta49) [inline pkg/crmanager/validate.go:95]: can we use tsResource.Spec.Type=="" as it is of type string.

Comment (sravyap135) [reply pkg/crmanager/validate.go]: Done

**Pull request merged**